string info
---
bool is_moving

